### DISCUSSION TITLE
Deploy Rowy Run

### DISCUSSION THREAD

**Discussion by eggwardi** [AntlerVC/firetable#1392] [Support / Q&A] [answered]
Hello, new here and trying to play around with Rowy.  When I deploy I get an error. My logs show the following:

who is iamstevepinto?

`{
  "protoPayload": {
    "@type": "type.googleapis.com/google.cloud.audit.AuditLog",
    "status": {
      "code": 7,
      "message": "Permission 'iam.serviceAccounts.create' denied on resource (or it may not exist).",
      "details": [
        {
          "@type": "type.googleapis.com/google.rpc.ErrorInfo",
          "reason": "IAM_PERMISSION_DENIED",
          "domain": "iam.googleapis.com",
          "metadata": {
            "permission": "iam.serviceAccounts.create"
          }
        }
      ]
    },
    "authenticationInfo": {
      "principalEmail": "iamstevepinto@gmail.com",
      "principalSubject": "user:iamstevepinto@gmail.com"
    },
    "requestMetadata": {
      "callerIp": "2600:1900:2000:e9::8",
      "callerSuppliedUserAgent": "google-api-nodejs-client/5.1.0 (gzip),gzip(gfe)",
      "requestAttributes": {
        "time": "2023-08-28T16:20:13.209862150Z",
        "auth": {}
      },
      "destinationAttributes": {}
    },
    "serviceName": "iam.googleapis.com",
    "methodName": "google.iam.admin.v1.CreateServiceAccount",
    "authorizationInfo": [
      {
        "resource": "projects/inspections-tbzotd",
        "permission": "iam.serviceAccounts.create",
        "resourceAttributes": {}
      }
    ],
    "resourceName": "projects/inspections-tbzotd",
    "request": {
      "account_id": "rowy-backend",
      "service_account": {
        "description": "Used in Rowy Backend cloud run service",
        "display_name": "Rowy Backend"
      },
      "name": "projects/inspections-tbzotd",
      "@type": "type.googleapis.com/google.iam.admin.v1.CreateServiceAccountRequest"
    },
    "response": {
      "@type": "type.googleapis.com/google.iam.admin.v1.ServiceAccount"
    }
  },
  "insertId": "1w9p25ieb6m24",
  "resource": {
    "type": "service_account",
    "labels": {
      "project_id": "inspections-tbzotd",
      "unique_id": "",
      "email_id": ""
    }
  },
  "timestamp": "2023-08-28T16:20:13.155836318Z",
  "severity": "ERROR",
  "logName": "projects/inspections-tbzotd/logs/cloudaudit.googleapis.com%2Factivity",
  "receiveTimestamp": "2023-08-28T16:20:15.125617814Z"
}`

**Comment by Chadha93** [comment, marked as answer]
A brand new release of Rowy was made last week, with lots of bug fixes and enhancements. 

https://github.com/rowyio/rowy/releases/tag/v3.0.0

Will you please try again? Let us know if this is still an issue. Closing this for now.
